### ISSUE TITLE
don't close on losing hover 

### ISSUE THREAD

**Issue by smartPangelous** [mohsinulhaq/react-popper-tooltip#145]
I am trying to change this official example to suits my project's needs :
(persist-once-mounted)
https://codesandbox.io/s/github/mohsinulhaq/react-popper-tooltip/tree/master/examples/persist-once-mounted?file=/src/index.js

On the above example the popper keeps open while the box is on hover and when losing hover it closes . How can I convert this example's behavior somehow that when user moves the pointer out of opened box, the popper stays open unless the user clicks out of the box ?

**Comment by denisborovikov** [comment]
It looks like a bug in our code. I'll look into this later.

**Comment by denisborovikov** [comment]
I think as a workaround you can remove `interactive: true` from the hook's arguments, copy our default CSS file (if you use it), and remove

```
.tooltip-container[data-popper-interactive='false'] {
  pointer-events: none;
}
```

**Comment by smartPangelous** [comment]
This just solved the problem. thanks

**Comment by SaalkT** [comment]
Hi, i dont use the default css file.
But this problem still occurs, How to fix this? 
A working 'interactive' flag would be nice :)

**Comment by mohsinulhaq** [comment]
This is now fixed in `v4.4.2`

**Comment by rahildar** [comment]
I added a delayHide of 100 ms in `usePopperTooltip` and it fixed it for me - as soon as I would hover out of trigger element the tooltip would close otherwise
